framework/include/mediaplayer/mediaplayer.h

```diff
@@ -0,0 +1,213 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __MEDIAPLAYER_H__
+#define __MEDIAPLAYER_H__
+
+/****************************************************************************
+* Included Files
+****************************************************************************/
+#include <queue.h>
+#include <sys/types.h>
+enum media_result_e {
+	MEDIA_OK,		
+	MEDIA_ERROR,
+	MEDIA_ERROR_THREAD_INIT,
+	MEDIA_ERROR_PLAYBACK,
+	MEDIA_ERROR_RECORD,
+	MEDIA_ERROR_WRITE_TO_FILE,
+	MEDIA_ERROR_CREATE,
+	MEDIA_ERROR_UNKNOWN_FILE
+};
+
+typedef enum media_result_e media_result_t;
+
+
+enum media_type_e {
+	MEDIA_TYPE_PCM,
+	MEDIA_TYPE_WAV
+};
+
+typedef enum media_type_e media_type_t;
+
+enum media_play_type_e {
```

Comment:
Duplication of "media" in enum tags and constants.

Reply:
Are you suggesting to remove all MEDIA_ from all enum definitions. In my opinion it is better to use media_ to make code more readable.